test/parallel/test-crypto-x509.js

```diff
@@ -359,3 +359,14 @@ UcXd/5qu2GhokrKU2cPttU+XAN2Om6a0
 
   assert.strictEqual(cert.checkIssued(cert), false);
 }
+
+{
+  // Test X509Certificate extension methods
+
+  const certPemWithExtensions = readFileSync(fixtures.path('keys', 'crypto-extensions.pem'));
+  const certWithExtensions = new X509Certificate(certPemWithExtensions);
+  const extensions = certWithExtensions;
+
+  assert(extensions.subjectAltName);
+  assert.strictEqual(extensions.subjectAltName, 'DNS:example.com, DNS:www.example.com');
+}
```

Comment:
This is not testing the extensions getter at all. 

In fact, I cannot see an underlying extensions functionality in the C++ code, or am I missing it?